packages/roosterjs-content-model/lib/editor/corePlugins/ContentModelCopyPastePlugin.ts

```diff
@@ -119,6 +123,11 @@ export default class ContentModelCopyPastePlugin implements PluginWithState<Copy
                 {
                     isDarkMode: false /* To force light mode on paste */,
                     darkColorHandler: this.editor.getDarkColorHandler(),
+                },
+                {
+                    modelHandlerOverride: {
+                        table: copyCutTableHandler,
```

Comment:
You can leverage the onNodeCreated parameter from ModelToDomOption. It will be called when a node is created, then you can check if it is a table then wrap it.

Using a customized handler here is fine, but using cachedElement is not a good idea here.

Reply:
There is an exmaple about how to use this parameter in packages\roosterjs-content-model\lib\publicApi\link\insertLink.ts